packages/drivers/odsp-driver/src/odspDocumentStorageManager.ts

```diff
@@ -261,29 +261,22 @@ export class OdspDocumentStorageService implements IDocumentStorageService {
             blob = response.content;
         }
 
+        let documentAttributes: api.IDocumentAttributes = JSON.parse(blob.content);
+        if (blob.encoding === "base64") {
```

Comment:
I'd assert that encoding is empty (not present) if it's not base64 (and make it optional on the interface).
Alternatively we can give it a name, "unicode" would be my preference (given [RFC 4627](https://tools.ietf.org/html/rfc4627) is clear that utf-8/utf-16/utf-32 (BE or LE) are all supported and auto-detected).